core/templates/hashfuncs.h

```diff
@@ -55,6 +55,10 @@
 #include "core/templates/rid.h"
 #include "core/typedefs.h"
 
+#ifdef _MSC_VER
+#include <intrin.h>
```

Comment:
For odd includes like this I usually like to add a comment, like `// needed for `__umulh` below`.